### DISCUSSION TITLE
sheet component animation is not smooth

### DISCUSSION THREAD

**Discussion by dinesh-hills** [hngngn/shadcn-solid#64] [Q&A] [answered]
@hngngn Did you notice the sheet animation is not smooth? I compared the shadcn-solidjs with original shadcn, original sheet is really smooth compared the current shadcn-solidjs sheet component.

**Comment by dinesh-hills** [comment]
I suppose, it's because you're adding a blur effect to the background?

**Reply by hngngn** [reply, marked as answer]
Check the new update. It's not smoother, though. But in the end, this is just a base style to help you get started. You can always edit it to match your needs.

**Reply by dinesh-hills** [reply]
Yeah, well a smooth component out of the box would be better I suppose.
